src/export/word/template.ts

```diff
@@ -38,6 +38,10 @@ export async function defaultTemplate(data: LoadedArticle): Promise<Document> {
     if (!state) return;
     docxState.renderContent(state.doc);
   });
+  Object.values(article.references).map(({ state }) => {
+    if (!state) return;
+    docxState.renderContent(state.doc);
+  });
```

Comment:
Some remaining tasks:
- [ ] Understand if the article order here will be deterministic and correct
- [ ] Need to a title for references if they exist (similar to `createArticleTitle` above?)
- [ ] Should we use `renderReference` from the [proposed prosemirror-docx](https://github.com/curvenote/prosemirror-docx/pull/9) changes? Probably doesn't matter here since there is no content between the main article content and the references (unlike custom templates)